app/controllers/api/base_controller/authentication.rb

```diff
@@ -7,10 +7,10 @@ module Authentication
       def require_api_user_or_token
         log_request_initiated
         @auth_token = @auth_user = nil
-        if request.headers['X-MIQ-Token']
-          authenticate_with_system_token(request.headers['X-MIQ-Token'])
-        elsif request.headers['X-Auth-Token']
-          authenticate_with_user_token(request.headers['X-Auth-Token'])
+        if request.headers[HttpHeaders::MIQ_TOKEN]
```

Comment:
It's in https://github.com/rails/rails/blob/67263020ec4c40ff1cef5080056b233e9d723116/actionpack/lib/action_dispatch/http/headers.rb#L118-L127 - Rails is quite flexible over how you reference those headers. We could have another set of constants, but this seemed to be the preferred solution if we can do both